app/javascript/articles/components/TagList.jsx

```diff
@@ -1,8 +1,9 @@
 import { h } from 'preact';
+import PropTypes from 'prop-types';
 import { tagPropTypes } from '../../src/components/common-prop-types';
 
-export const TagList = ({ tags = [] }) => (
-  <div className="tags">
+export const TagList = ({ tags = [], className }) => (
+  <div className={`tags ${className || ''}`}>
```

Comment:
This is to support the `<FeaturedArticle />` component which takes additional "featured" classes.